php/WP_CLI/Configurator.php

```diff
@@ -324,17 +324,17 @@ private static function load_yml( $yml_file ) {
 		}
 
 		// Backwards compat
-		// 'core config' -> 'config create'
+		// Index 'core config' was renamed to 'config create'.
```

Comment:
These comments are not directly referring to the indexdes here (although this is what needs to be adapted), they are referring to the actual commands, as they are expressed as strings.
```suggestion
		// Command 'core config' was moved to 'config create'.
```